crates/bevy_winit/src/web_resize.rs

```diff
@@ -0,0 +1,83 @@
+use crate::WinitWindows;
+use bevy_app::{App, Plugin};
+use bevy_ecs::prelude::*;
+use bevy_window::WindowId;
+use crossbeam_channel::{Receiver, Sender};
+use wasm_bindgen::JsCast;
+use winit::dpi::LogicalSize;
+
+pub struct CanvasParentResizePlugin;
```

Comment:
Docs please; this is public. In particular, this should link to [`WindowDescriptor`].

Reply:
Given that this plugin is automatically added / just an implementation detail / relies on internals to work properly, I've opted to make this `pub(crate)` instead. The resource was already private.